packages/integration-tests/src/EntityManager.test.ts

```diff
@@ -1125,6 +1125,17 @@ describe("EntityManager", () => {
     // Then we expect the cloned entity to have no references
     expect(await a2.books.load()).toHaveLength(0);
   });
+
+  it("can touch an entity to force it to be flushed", async () => {
+    await insertAuthor({ first_name: "a1" });
+    const em = newEntityManager();
+    const a1 = await em.load(Author, "1");
+    em.touch(a1);
+    expect(a1["__isTouched"]).toBeTruthy();
+    await em.flush();
+    expect(a1.beforeFlushRan).toBeTruthy();
```

Comment:
Should we assert that other things like derived fields ran? I'm fine trusting it that it does happen. Just if it'd be easy.

Reply:
I don't think that's necessary.  Actually think i might change this to just check the return of the flush, since it returns `entitiesToFlush`.